infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServerCache.java

```diff
@@ -32,13 +38,21 @@
     //Empirical threshold of number of streams in a logdata beyond which server performance may be slow
     private final int MAX_STREAM_THRESHOLD = 20;
 
+    private final Optional<Timer> readTimer;
```

Comment:
Can you also capture stats on the hit ratio, load time and total weight.

Reply:
Added